FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataProcessor/DataConverter/CustomSchemaConverter.cs

```diff
@@ -45,8 +45,8 @@ public class CustomSchemaConverter : IDataSchemaConverter
         {
             EnsureArg.IsNotNull(schemaConfiguration, nameof(schemaConfiguration));
 
-            _diagnosticLogger = EnsureArg.IsNotNull(diagnosticLogger, nameof(diagnosticLogger));
             _logger = EnsureArg.IsNotNull(logger, nameof(logger));
```

Comment:
Is it possile that `JosnProcessor()` throw an exception? do we need lazy initialization for it?